src/App.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useState } from 'react';
+import UserHeader from './components/header';
+import CharacterCard from './components/characters';
+import { ContentStyle } from './styled/style';
+import { characters } from './data';
+
+type Character = {
+  id?: number;
+  image: string;
+  name: string;
+}
+
+function App() {
```

Comment:
use arrow function here and add typescript typing

`const App:React.FC = () => {`